{{cookiecutter.beat}}/tests/system/config/{{cookiecutter.beat}}.yml.j2

```diff
@@ -0,0 +1,83 @@
+################### Beat Configuration #########################
+
+
+
+############################# Output ##########################################
+
+# Configure what outputs to use when sending the data collected by the beat.
+# You can enable one or multiple outputs by setting enabled option to true.
+output:
+
+  ### File as output
+  file:
+    # Enabling file output
+    enabled: true
+
+    # Path to the directory where to save the generated files. The option is mandatory.
+    path: {{ output_file_path|default(beat.working_dir + "/output") }}
+
+
+    # Name of the generated files. The default is `topbeat` and it generates
+    # files: `topbeat`, `topbeat.1`, `topbeat.2`, etc.
+    filename: "{{ output_file_filename|default("topbeat") }}"
```

Comment:
Normally yes, the problem is that I exclude this file from cookiecutter as it least to a lot of other issue because it is also a template. So perhaps we just go with "beat" here.